test_util/aws.py

```diff
@@ -349,10 +333,11 @@ def delete(self):
         self.stack = self.boto_wrapper.resource('cloudformation').Stack(self.stack.stack_id)
         self.stack.delete()
 
+    @retry_boto_rate_limits
     def get_vpc_host_ips(self):
         reservations = self.boto_wrapper.client('ec2').describe_instances(Filters=[{
```

Comment:
is it feasible to build this into the boto_wrapper for ec2 perhaps rather than having to remember to hand-wrap every usage of it?